chain/network/src/codec.rs

```diff
@@ -74,12 +74,63 @@ pub fn bytes_to_peer_message(bytes: &[u8]) -> Result<PeerMessage, std::io::Error
     PeerMessage::try_from_slice(bytes)
 }
 
+fn peer_id_type_field_len(enum_var: u8) -> Option<usize> {
+    // 1 byte for enum variant, then some number depending on the
+    // public key type
+    match enum_var {
+        0 => Some(1 + 32),
+        1 => Some(1 + 64),
+        _ => None,
+    }
+}
+
+pub fn is_forward_tx(bytes: &[u8]) -> bool {
+    // PeerMessage::Routed variant == 13
+    if bytes[0] == 13 {
```

Comment:
Where do we check `bytes` length?

Reply:
good point.

Reply:
Good catch! Addressed in https://github.com/nearprotocol/nearcore/pull/3317/commits/1af9bf0cbf66e19808283fe9bcd51e1196ff1fb0